src/main/java/frc/team2412/robot/commands/intake/IntakeBallCommand.java

```diff
@@ -0,0 +1,21 @@
+package frc.team2412.robot.commands.intake;
+
+import edu.wpi.first.wpilibj2.command.ParallelCommandGroup;
+import frc.team2412.robot.commands.index.IndexCommand;
+import frc.team2412.robot.subsystem.IndexSubsystem;
+import frc.team2412.robot.subsystem.IntakeSubsystem;
+
+public class IntakeBallCommand extends ParallelCommandGroup {
+    private final IndexSubsystem indexSubsystem;
+    private final IntakeSubsystem intakeSubsystem;
+
+    public IntakeBallCommand(IndexSubsystem indexSubsystem, IntakeSubsystem intakeSubsystem) {
+        this.indexSubsystem = indexSubsystem;
+        this.intakeSubsystem = intakeSubsystem;
+        addRequirements(indexSubsystem, intakeSubsystem);
```

Comment:
Technically, calling `addRequirements()` isn't needed (for this command and `SpitBallCommand`), because it's done automatically for command groups (see the [wpilib documentation](https://docs.wpilib.org/en/stable/docs/software/commandbased/command-groups.html)). Calling it explicitly won't break anything, though.